content/id/docs/setup/learning-environment/minikube.md

```diff
@@ -0,0 +1,515 @@
+
+_Flag_ ini menerima sebuah _string_ dalam format `component.key=value`, dimana `component` adalah salah satu _string_ dari list dibawah, `key` adalah nilai dari _struct_ configurasi dan `value` adalah nilai yang digunakan.
+
+Kunci yang valid bisa ditemukan dengan memeriksa dokumentasi `componentconfigs` Kubernetes untuk setiap komponen.
+Berikut adalah dokumentasi untuk setiap konfigurasi yang didukung:
+
+* [kubelet](https://godoc.org/k8s.io/kubernetes/pkg/kubelet/apis/config#KubeletConfiguration)
+* [apiserver](https://godoc.org/k8s.io/kubernetes/cmd/kube-apiserver/app/options#ServerRunOptions)
+* [proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+* [controller-manager](https://godoc.org/k8s.io/kubernetes/pkg/controller/apis/config#KubeControllerManagerConfiguration)
+* [etcd](https://godoc.org/github.com/coreos/etcd/etcdserver#ServerConfig)
+* [scheduler](https://godoc.org/k8s.io/kubernetes/pkg/scheduler/apis/config#KubeSchedulerConfiguration)
+
+#### Contoh
+
+Untuk mengubah pengaturan `MaxPods` menjadi 5 pada Kubelet, gunakan _flag_ ini: `--extra-config=kubelet.MaxPods=5`.
+
+Fitur ini juga mendukung _struct_ bersarang. Untuk mengubah pengaturan `LeaderElection.LeaderElect` menjadi `true` pada penjadwal, gunakan _flag_: `--extra-config=scheduler.LeaderElection.LeaderElect=true`.
+
+Untuk mengatur `AuthorizationMode` pada `apiserver` menjadi `RBAC`, kamu bisa menggunakan: `--extra-config=apiserver.authorization-mode=RBAC`.
+
+### Menghentikan klaster
+Perintah `minikube stop` bisa digunakan untuk menghentikan klaster kamu.
+Perintah ini menghentikan mesin virtual Minikube, tapi mempertahankan semua _state_ dan data klaster.
+Memulai klaster lagi akan mengembalikannya ke _state_ sebelumnya.
+
+### Menghapus klaster
+Perintah `minikube delete` bisa digunakan untuk menghapus klaster kamu.
+Perintah ini menghentikan dan menghapus mesin virtual Minikube. Tidak ada data atau _state_ yang dipertahankan.
+
+### Memperbaharui Minikube
+Jika kamu menggunakan macOS dan [Brew Package Manager](https://brew.sh/) sudah terpasang, jalankan:
+
+```shell
+brew update
+brew upgrade minikube
+```
+
+## Interaksi dengan Klaster Kamu
+
+### Kubectl
+
+Perintah `minikube start` membuah sebuah [konteks kubectl](/docs/reference/generated/kubectl/kubectl-commands#-em-set-context-em-) yang disebut "minikube".
+Konteks ini menyimpan pengaturan untuk berkomunikasi dengan klaster Miniku kamu.
+
+Minikube menetapkan konteks ini sebagai bawaan secara otomatis, tetapi jika kamu ingin mengubah kembali ke konteks tersebut di kemudian hari, gunakan:
+
+`kubectl config use-context minikube`
+
+Atau berikan konteks untuk setiap perintah seperti ini:
+
+`kubectl get pods --context=minikube`
+
+### Dashboard
+
+Untuk mengakses [Dashboard Kubernetes](/docs/tasks/access-application-cluster/web-ui-dashboard/), gunakan perintah ini pada terminal setelah memulai Minikube untuk mendapatkan alamatnya:
```

Comment:
```suggestion
Untuk mengakses [Dasbor Kubernetes](/id/docs/tasks/access-application-cluster/web-ui-dashboard/), gunakan perintah ini pada terminal setelah memulai Minikube untuk mendapatkan alamatnya:
```

Reply:
`Kubernetes Dashboard` is the proper name of the software, I wonder if we can leave it as is instead 🤔